src/preset.ts

```diff
@@ -1180,7 +1055,64 @@ async function tryApplyVsDevEnv(preset: ConfigurePreset, workspaceFolder: string
     preset.__parentEnvironment = EnvironmentUtils.mergePreserveNull([process.env, developerEnvironment]);
 }
 
-async function expandConfigurePresetHelper(folder: string, preset: ConfigurePreset, allowUserPreset: boolean = false, errorHandler?: ExpansionErrorHandler) {
+export async function expandConfigurePreset(folder: string, name: string, workspaceFolder: string, sourceDir: string, allowUserPreset: boolean = false, enableTryApplyDevEnv: boolean = true, errorHandler?: ExpansionErrorHandler): Promise<ConfigurePreset | null> {
```

Comment:
Similar comment here: I think we should add a comment for the parameters explaining the context and what they are used for, especially the `enableTryApplyDevEnv`. 

Also, correct me if I'm wrong, but I think the default for the `enableTryApplyDevEnv` could probably be `false` now right? 
Since it should only be true when we're actually setting the configure preset and re-expanding? It's just a default so as long as we set it correctly at invocation time, it's fine either way, just make sure it's documented, whichever direction you pick 👍

Reply:
added comment and yeah I set it to be false